### ISSUE TITLE
minitalk tests not working 

### ISSUE THREAD

**Issue by UncleJ4ck** [xicodomingues/francinette#29]
Bug:

when i start the francinette in my root projecs, it keeps working normally until it starts the tests and it displays an error about leaks "Leaks: ⠙'NoneType' object has no attribute 'group'"

**Comment by Kuninoto** [comment]
Yep. I'm having the same bug!

**Comment by Kuninoto** [comment]
@xicodomingues Yes! It is available on my repository: https://github.com/Kuninoto/42_minitalk

**Comment by xicodomingues** [comment]
This specific error does not happen to me. Maybe it is related to operating system.
I'm on MacOS Catalina. I can try and test it on linux.

What occurs to me is that `grep` may work differently on Linux than on mac. (I depend on it to know which files have a `main`)

@Kuninoto
In your case, that error does not happen on my pc, but what happens is that because you do not terminate the `client` after sending a message then the test times out, because the tester depends on that behavior.

**Comment by fhassoun** [comment]
i have the same problem, it gives me : 
✔ Executing: make fclean bonus
ℹ Running tests: fsoares (my own tests)
Leaks: ⠙'NoneType' object has no attribute 'group'

im on ubuntu 20.04
